packages/documentation/src/stories/components/list/list.docs.mdx

```diff
@@ -0,0 +1,29 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+import * as ListStories from './list.stories';
+
+<Meta of={ListStories} />
+
+<div className="docs-title">
+  # List
+
+  <nav>
+    <link-design of={JSON.stringify(ListStories)}></link-design>
+  </nav>
+</div>
+
+The `<post-list>` is a container for `<post-list-item>` components.
+
+<Canvas sourceState="shown" of={ListStories.Default} />
+
+## `<post-list>`
+
+<Controls of={ListStories.Default} />
+
+## `<post-list-item>`
```

Comment:
Since the post-list-item has no properties, this section is not really necessary.